relay-server/src/services/processor/event.rs

```diff
@@ -271,15 +271,35 @@ pub fn finalize<G: EventProcessing>(
     Ok(())
 }
 
+/// Decision for applying some filters that don't drop the event.
```

Comment:
Bikeshedding a bit: I don't really like "decision" here, but the alternatives "result" and "outcome" have other meanings that may generate some confusion. Open to suggestions.